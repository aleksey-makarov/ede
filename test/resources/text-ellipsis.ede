{
  "short": "short-text",
  "long": "A slightly longer piece of text that would like to be truncated."
}
---
{{ short | toEllipsis(5) }}
{{ short | toEllipsisWith(2, "0xb") }}
{{ long | toEllipsis(20) }}
{{ long | toEllipsisWith(20, " -") }}
{{ long | toEllipsis(255) }}
