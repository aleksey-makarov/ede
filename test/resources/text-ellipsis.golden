short...
sh0xb
A slightly longer pi...
A slightly longer pi -
A slightly longer piece of text that would like to be truncated.
